README.md

```diff
@@ -1 +1,10 @@
 # community
+
+![coala Header](/docs/_static/images/coala-header.png)
+
+## About
+
+* This repository is a static django website;
```

Comment:
The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*